sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1119,6 +1269,151 @@ export class BlobClient extends StorageClient {
+   *                        identity values returned in the x-ms-owner, x-ms-group, and x-ms-acl response headers will be
+   *                        transformed from Azure Active Directory Object IDs to User Principal Names.  If "false", the
+   *                        values will be returned as Azure Active Directory Object IDs. The default value is false.
+   * @param {BlobGetAccessControlOptions} [options={}]
+   * @returns {Promise<Models.BlobGetAccessControlResponse>}
+   * @memberof BlobClient
+   */
+  public async getAccessControl(
+    upn?: boolean,
+    options: BlobGetAccessControlOptions = {}
+  ): Promise<Models.BlobGetAccessControlResponse> {
+    options.blobAccessConditions = options.blobAccessConditions || {};
+    const aborter = new ContextAborter(options.abortSignal || AbortSignal.none, DfsContext);
+    return this.blobContext.getAccessControl({
+      abortSignal: aborter,
+      upn,
+      leaseAccessConditions: options.blobAccessConditions.leaseAccessConditions,
+      modifiedAccessConditions: options.blobAccessConditions.modifiedAccessConditions
+    });
+  }
+
+  /**
+   * Rename current blob to another blob. By default, the destination is overwritten and if the destination already
+   * exists and has a lease the lease is broken. This operation supports conditional HTTP requests.
+   * For more information, see [Specifying Conditional Headers for Blob Service
+   * Operations](https://docs.microsoft.com/en-us/rest/api/storageservices/specifying-conditional-headers-for-blob-service-operations).
+   * To fail if the destination already exists, use a conditional request with If-None-Match: "*".
+   *
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/create
+   *
+   * @param {BlobClient} destination Destination `BlobClient` object. Must be same storage account with source.
+   * @param {BlobMoveOptions} [options={}]
+   * @returns {Promise<Models.BlobRenameResponse>}
+   * @memberof BlobClient
+   */
+  public async move(
+    destination: BlobClient,
+    options: BlobMoveOptions = {}
+  ): Promise<Models.BlobRenameResponse> {
+    options.sourceAccessConditions = options.sourceAccessConditions || {};
+    options.sourceAccessConditions.leaseAccessConditions =
+      options.sourceAccessConditions.leaseAccessConditions || {};
+    options.sourceAccessConditions.modifiedAccessConditions =
+      options.sourceAccessConditions.modifiedAccessConditions || {};
+    options.destAccessConditions = options.destAccessConditions || {};
+
+    if (getURLHost(this.url) !== getURLHost(destination.url)) {
+      throw RangeError(`Destination BlobClient must be in the same storage account with source.`);
+    }
+
+    const source = options.sourcePath || getURLPath(this.url);
+    if (source === undefined) {
+      throw RangeError(`Invalid move source path ${source}`);
```

Comment:
Minor: Missing . at end? Making forma consistent would be good.

Reply:
Good catch.

Reply:
good catch

---
In reply to: [318944216](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318944216) [](ancestors = 318944216)